### ISSUE TITLE
cluster.routing.allocation.disk.threshold_enabled default set to false

### ISSUE THREAD

**Issue by vincenzodnp** [uselagoon/lagoon#1571]
**Describe the bug**
Elasticsearch, as reported in the official documentation, has a `disk allocation decider` which controls where allocate new shards or where relocate shards from a node in case of disk usage above a threshold.
The default value in ES 7.5 is `true` but for some reason, Opendistro sets it to `false`.

```
grep cluster.routing.allocation.disk.threshold_enabled /usr/share/elasticsearch/config/elasticsearch.yml 
cluster.routing.allocation.disk.threshold_enabled: false
```

**To Reproduce**
1. Deploy Elasticsearch
2. Grep cluster.routing.allocation.disk.threshold_enabled  on the configuration file

**Expected behavior**
`cluster.routing.allocation.disk.threshold_enabled`  set to `true`

**Comment by Schnitzel** [comment]
I closed #1580 as there is now support for ES 7.6.1 on OpenDistro: https://opendistro.github.io/for-elasticsearch-docs/docs/install/plugins/

Let's upgrade our ES to 7.6.1 in order to stay up to date.

**Comment by tobybellwood** [comment]
Opendistro/Opensearch is no longer controlled by Lagoon
